pydicom/dataelem.py

```diff
@@ -166,7 +172,14 @@ def __init__(self,
             tag = Tag(tag)
         self.tag = tag
         self.VR = VR  # Note!: you must set VR before setting value
-        if already_converted:
+        if value is None:
+            if VR in INT_VR_VALUES:
+                self._value = MultiValue(int, [])
```

Comment:
I'm not sure if this is the best way to do this since it fails the reflexive test

```python
ds.SomeElement = None
assert ds.SomeElement == None
```

Reply:
I agree with the reflexive test here, anything set to None should ideally be kept as None, if that is possible.

Reply:
I partly agree. `None` is a natural value representation for empty type 2 elements, so using it instead of empty arrays makes sense. For numerical elements this should probably be the returned value. For text values, I'm not so sure, as they can also be represented by an empty string - there is no difference between an empty value and no value, so we have to decide what to use, and I tend to use an empty string as being less disruptive. It is a possibility to adapt comparisons to return True both for empty strings and `None`, though I'm not sure about that either.
Also keep in mind that after saving and reloading a dataset we cannot know if we originally assigned an empty string, an empty array or `None` to a text element, as they are all equivalent - we have to decide for a default representation. I may have to think more about this...

Reply:
I partly agree. `None` is a natural value representation for empty type 2 elements, so using it instead of empty arrays makes sense. For numerical elements this should probably be the returned value. For text values, I'm not so sure, as they can also be represented by an empty string - there is no difference between an empty value and no value, so we have to decide what to use, and I tend to use an empty string as being less disruptive. It is a possibility to adapt comparisons to return True both for empty strings and `None`, though I'm not sure about that either.
Also keep in mind that after saving and reloading a dataset we cannot know if we originally assigned an empty string, an empty array or `None` to a text element, as they are all equivalent - we have to decide for a default representation. I may have to think more about this...